packages/admin/cms-admin/src/dam/FileForm/FileSettingsFields.tsx

```diff
@@ -27,14 +27,12 @@ const damIsFilenameOccupiedQuery = gql`
 
 export type LicenseType = GQLLicenseType | "NO_LICENSE";
 
-const licenseTypeArray: readonly LicenseType[] = ["NO_LICENSE", "ROYALTY_FREE", "RIGHTS_MANAGED", "SUBSCRIPTION", "MICRO"];
+const licenseTypeArray: readonly LicenseType[] = ["NO_LICENSE", "ROYALTY_FREE", "RIGHTS_MANAGED"];
 
-const licenseTypeLabels: { [key in LicenseType]: React.ReactNode } = {
+const licenseTypeLabels: { [key in LicenseType]?: React.ReactNode } = {
```

Comment:
`?` shouldn't be necessary if you have all keys

```suggestion
const licenseTypeLabels: { [key in LicenseType]: React.ReactNode } = {
```